airflow/contrib/hooks/spark_sql_hook.py

```diff
@@ -98,10 +98,11 @@ def _prepare_command(self, cmd):
         if self._num_executors:
             connection_cmd += ["--num-executors", str(self._num_executors)]
         if self._sql:
-            if self._sql.endswith('.sql') or self._sql.endswith(".hql"):
-                connection_cmd += ["-f", self._sql]
+            sql = self._sql.strip()
+            if sql.endswith('.sql') or sql.endswith(".hql"):
```

Comment:
nit: should use either single or double quotes in both places (considering the rest of the file I'd personallly prefer double)